### PR TITLE
Send separate emails during alerts

### PR DESCRIPTION
**Pull request by siminm**
cc: @diranged

### CODE REVIEW

File: zk_monitor/alerts/email.py [hunk 1]
```diff
@@ -74,12 +74,16 @@ def _alert(self, path, state, message, params):
...
-            body=body,
-            email=to_address,
-            conn=self._mail_backend)
+
+        if type(emails) == str:
+            emails = re.compile('[, ]+').split(emails)
+
+        for e in emails:
+            log.debug('Creating Email Alert Message: %s to %s' % (message, e))
+            EmailAlert(subject=subject,
```
Comment (diranged) [inline zk_monitor/alerts/email.py]: Quick question while we're in here.. I'm concerned that if the `EmailAlert()` raises an exception, that `_alert()` method will re-raise the exception all the way up and prevent Tornado from finishing up the other alerters.

Can you dig into the code and see if this is the case? If so, we should patch that up while we're here to make sure that all of the alerts fire off, regardless of whether one fails or not.

**Comment by diranged** [comment]
:+1: except for my comment. can you dig into it?

**Comment by siminm** [comment]
Good point -- will try/catch and log a `critical()`

On Tue, Jun 9, 2015 at 1:28 PM, Matt W <notifications@github.com> wrote:

> [image: :+1:] except for my comment. can you dig into it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Nextdoor/zkmonitor/pull/29#issuecomment-110495240>.
>



-- 
Mikhail Simin, Ph.D
*Nextdoor* <http://nextdoor.com/>
The Private Social Network for Neighborhoods

**Comment by siminm** [comment]
More digging is required. Starting to suspect that my change won't fix the issue.

**Pull request merged**